js/index.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable camelcase */
+import { SlideDeck } from './slidedeck.js';
+
+const map = L.map('map', { scrollWheelZoom: false }).setView([0, 0], 0);
```

Comment:
**suggestion:** Here's a sneaky pro-tip to get smoother zooming on your polygons:
```suggestion
const SmoothRenderer = L.Canvas.extend({
  // Override the default _onZoom function, which simply scales the lower
  // resolution shapes. Instead, we want to reproject the shapes at each new
  // zoom level, as is done by default when zooming ends.
  _onZoom: function() {
    this._onZoomEnd();
    this._update();
  },
});

const map = L.map('map', { scrollWheelZoom: false, renderer: new SmoothRenderer() }).setView([0, 0], 0);
```
This will override the default renderer (i.e. the built-in Leaflet SVG renderer) with one that redraws the your shapes at each step of a fly-in instead of just at the end. Note, this will work best if you simplify your polygons as much as reasonable, as I mentioned above.